gschem/src/x_image.c

```diff
@@ -129,7 +129,7 @@ x_image_get_type_from_description (const char *description)
   GSList *ptr;
 
   if (strcmp (description, "Portable Document Format") == 0) {
-    return "pdf";
+    return (char*) "pdf";
```

Comment:
@peter-b I'll better add another PR to solve these two cast issues. Thank you for your review and for your patience with me!